WordPress/src/main/java/org/wordpress/android/ui/posts/PostUtils.java

```diff
@@ -383,4 +387,34 @@ function gutenberg_content_has_blocks( $content ) {
     public static boolean contentContainsGutenbergBlocks(String postContent) {
         return (postContent != null && postContent.contains(GUTENBERG_BLOCK_START));
     }
+
+    public static void showGutenbergCompatibilityWarningDialog(Context ctx,
+                                                               FragmentManager fragmentManager,
+                                                               PostModel post,
+                                                               SiteModel site) {
+        GutenbergWarningFragmentDialog gutenbergCompatibilityDialog = new GutenbergWarningFragmentDialog();
+        gutenbergCompatibilityDialog.initialize(TAG_GUTENBERG_CONFIRM_DIALOG, post.isPage(), post.getRemotePostId());
+        gutenbergCompatibilityDialog.show(fragmentManager, TAG_GUTENBERG_CONFIRM_DIALOG);
+
+        // track event
+        Map<String, Object> properties = new HashMap<>();
```

Comment:
Can we re-use the `trackGutenbergDialogEvent()` method defined in the same file below?

Reply:
heh for sure, done in 7aa01612e3